docs/API_reference.md

```diff
@@ -580,6 +581,20 @@ Return a POST request JSON response, if one of the requests fails return the err
 If the **value_id** field is not filled the method will return: `{'error': 'missing_parameter'}`.
 In case the language within the `lang` field of the JSON is invalid the method will return `{'error': 'invalid_language'}`.
 
+#### plenty_api_create_property_selection_name <a name=create-property-selection></a>
+
+Create a new name for an existing property selection value for the given language.
+
+[*Required parameter*]:
+
+The **property_id** parameter contains the property Id which a selection name will be created for, the **lang** parameter contains a two letter abbreviation of the target language (for a list of valid values look here: [Language codes](https://developers.plentymarkets.com/en-gb/developers/main/rest-api-guides/getting-started.html#_language_codes)), and the **name** parameter contains the visible name for the selection name in the given language.
```

Comment:
s/contains the property Id which a selection name will be created for/
  contains the Plentymarkets ID of a property for which a new selection name value is being added/